app/localization/en.json

```diff
@@ -2,16 +2,16 @@
   "newScorecard": "New Scorecard",
   "savedScorecard": "Saved Scorecard",
   "setting": "Setting",
-  "backendUrl": "Backend URL",
+  "serverKhmerUrl": "Server KhmerURL",
```

Comment:
@limkimsan we used server_url but not server_khmer_url, just it's localize depends on the locale(can be en or km)

Reply:
So it should "Server URL", right?

Reply:
yes, that is "Server URL"